src/compiler/html/inline-esm-import.ts

```diff
@@ -59,7 +59,8 @@ export const optimizeEsmImport = async (
       if (results.orgImportPaths.length > 0) {
         // insert inline script
         script.removeAttribute('src');
-        script.innerHTML = results.code;
+        // the `'\n'` is necessary here to avoid CI failures on browserstack
+        script.innerHTML = results.code + '\n';
```

Comment:
in case we ever remove BS, can we state that this prevents a potential HTML parsing issue with inlined JS l source map comments?